triagebot.toml

```diff
@@ -203,6 +208,38 @@ trigger_files = [
     "compiler/rustc_macros/src/query.rs"
 ]
 
+[autolabel."A-testsuite"]
+trigger_files = [
+    "src/test",
```

Comment:
Can we remove this from the autolabel? This causes basically every compiler PR to be labeled A-testsuite, since almost every compiler PR has an associated UI test with it.

Reply:
Sure, makes sense. Not sure when I'll have time though.